e2e/run.py

```diff
@@ -204,3 +206,12 @@ def test_prompt_manager_no_model_params():
 
     with mgr.exec() as prompt:
         assert prompt.params is None
+
+
+def test_prompt_manager_no_model_params_undeployed():
+    mgr = UsedByCiDontDeleteNoParamsUndeployedPromptManager(
+        UsedByCiDontDeleteNoParamsUndeployedMinorVersion.DANGEROUSLY_USE_UNDEPLOYED,
+    )
+
+    with mgr.exec() as prompt:
+        assert prompt.params is None
```

Comment:
Any reason why we don't have these tests under `tests` folder in this directory?

Reply:
yeah we could move them there, we'd just need to refactor them a bit to be pytest-compatible. we'd also need to update the pytest config so that `poetry run pytest` ignores the `tests/e2e/` folder by default. i can create a ticket for this